src/VisualStudio/LiveShare/Impl/Shims/TypeScriptHandlerShims.cs

```diff
@@ -180,15 +197,8 @@ internal class TypeScriptFindImplementationsHandlerShim : FindImplementationsHan
         public TypeScriptFindImplementationsHandlerShim(IThreadingContext threadingContext)
             => _threadingContext = threadingContext;
 
-        public async Task<object?> HandleAsync(TextDocumentPositionParams request, RequestContext<Solution> requestContext, CancellationToken cancellationToken)
-        {
-            // The return definition for TextDocumentImplementationName is SumType<Location, Location[]>.
-            // However Live Share is unable to handle a SumType return when using ILspRequestHandler.
-            // So instead we just return the actual value from the SumType.
-            // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1059193 tracks the fix.
```

Comment:
This was just fixed now?

Reply:
This was fixed by not returning sumtype when we didn't need to.
The original bug is not fixed if we returned sum type though, but since
a)  we're moving away from the liveshare integration point all together
b)  typescript is getting rid of this to move to lsp as well
c)  I shouldn't have been returning sumtype here anyway

I don't think it's super critical to track